queue/sqs_queue.js

```diff
@@ -13,7 +13,7 @@ AWS.config = {
 }
 let sqs = new AWS.SQS();
 const Consumer = require('sqs-consumer');
-const { insertTweet, insertALotOfTweets, insertFollow } = require('../database/dbHelpers.js');
+const { insertTweet, insertALotOfTweets, insertFollow, getFeed } = require('../database/dbHelpers.js');
```

Comment:
Not knowing much about the SQS and AWS, this all feels logical and reasonable, well-named, and easy to follow.